src/Str.php

```diff
@@ -34,6 +40,10 @@ public static function camel($subject)
      */
     public static function startsWith($subject, $start, $caseSensitive = true)
     {
+        if (empty($subject) || empty($start)) {
```

Comment:
> empty($start)

This has the side effect that `startsWith('foo', '')` returns `false` now.

What about `$subject = $subject ?? '';` and the same for `$start`?

Reply:
Just wanted to avoid unnecessary `strncasecmp(),substr()` calls when `$subject` or `$start` is empty.